pkg/federate/kubefed/cluster.go

```diff
@@ -181,7 +181,7 @@ func (f *federator) OnUpdate(oldObj, newObj interface{}) {
 	}
 }
 
-func (f *federator) buildFederatedClusterConfig(kubeFedCluster *fedv1.KubeFedCluster) (*rest.Config, error) {
+func (f *Federator) buildFederatedClusterConfig(kubeFedCluster *fedv1.KubeFedCluster) (*rest.Config, error) {
```

Comment:
This will change in subsequent patches/PRs?

Reply:
The kubefed implementation won't change - in fact we'll likely end up removing at some point.